user_tools/src/spark_rapids_pytools/rapids/qualification.py

```diff
@@ -488,8 +488,9 @@ def _process_custom_args(self):
         # This is noise to dump everything
         # self.logger.debug('%s custom arguments = %s', self.pretty_name(), self.ctxt.props['wrapperCtx'])
 
-    def __is_savings_calc_enabled(self):
-        return self.ctxt.get_ctxt('enableSavingsCalculations')
+    def __is_savings_calc_enabled(self) -> bool:
+        cost_savings_func_flag = self.ctxt.get_value('sparkRapids', 'cli', 'defaults', 'costSavingsSettings')
+        return cost_savings_func_flag and self.ctxt.get_ctxt('enableSavingsCalculations')
```

Comment:
Why is the flag not: `self.ctxt.get_value('sparkRapids', 'cli', 'defaults', 'costSavingsSettings', 'enabled')`?

Reply:
Oops my bad. Was a typo when I was testing.
Fixed it.